dockerfiles/onnxruntime-tensorrt.dockerfile

```diff
@@ -7,21 +7,37 @@
 # Dockerfile to run ONNXRuntime with TensorRT integration
 
 # nVidia TensorRT Base Image
-FROM nvcr.io/nvidia/tensorrt:20.07.1-py3
+FROM nvcr.io/nvidia/tensorrt:20.07.1-py3 AS runtime
 
 ARG ONNXRUNTIME_REPO=https://github.com/Microsoft/onnxruntime
 ARG ONNXRUNTIME_BRANCH=v1.5.3
 
-RUN apt-get update &&\
-    apt-get install -y sudo git bash unattended-upgrades
-
 WORKDIR /code
 ENV PATH /usr/local/nvidia/bin:/usr/local/cuda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:/opt/miniconda/bin:${PATH}
 ENV LD_LIBRARY_PATH /opt/miniconda/lib:$LD_LIBRARY_PATH
 
+RUN DEBIAN_FRONTEND=noninteractive \
```

Comment:
the `DEBIAN_FRONTEND` variable in here doesn't really affect anything really, as you can see from the build output.
I think the better usage in docker is to put it in `ARG` statement like the one in main dockerfile.